pages/learn/glossary.md

```diff
@@ -117,7 +130,7 @@ For example, the `not` keyword takes a subschema value and inverts its result, s
 Some subschemas may appear in more complex nested locations within a parent schema.
 The `allOf` keyword, for instance, takes an array of multiple subschemas and succeeds whenever all of the subschemas do individually.
 
-Whether something that otherwise *appears* to be a schema (based on its contents) actually *is* a subschema can be misleading at first glance without context or knowlege about its location within the parent schema.
+Whether something that otherwise *appears* to be a schema (based on its contents) actually *is* a subschema can be misleading at first glance without context or knowledge about its location within the parent schema.
```

Comment:
The spelling fix is appreciated, but the wording here is weird.  Can we clean this up while we're here?